python/tchannel/tornado/message_factory.py

```diff
@@ -0,0 +1,98 @@
+from ..messages import Types
+from ..exceptions import StreamingException
+from ..messages.common import FlagsType
+from ..messages.call_continue import CallContinueMessage
+
+
+class MessageFactory(object):
+
+    # 64KB Max frame size
+    # 16B (size:2 | type:1 | reserved:1 | id:4 | reserved:8)
+    # 1 2 Bytes can represent 0~64KB-1
+    MAX_PAYLOAD_SIZE = 0xFFEF   # 64*1024 - 16 - 1
```

Comment:
Nit: This should probably be expressed as `(2^16- 1) - 16` in the comment?

Even better might be to calculate it at import time like so,

    2**16 - 1 - frame_rw.size_rw.width() - frame_rw.header_rw.width()

Thoughts?